README.md

```diff
@@ -925,6 +925,59 @@ Back endpoint responsible for fetching and sending information to the database r
 `api/wellness` Sends an answer boolean to the database that specifies whether a student is symptomatic or not: true = symptomatic, false = not symptomatic.
 
 
+### Jobs
+Back endpoint responsible for fetching and sending information to the database regarding Jobs and timesheet used to clock shifts.
+
+##### GET 
+
+`api/jobs/getJobs` Gets the users current active jobs and returns them in a list.  
+
+`api/jobs/getSavedShifts/` Gets the users active shifts that have yet to be approved.
+
+`api/jobs/supervisorName/{supervisorID}`Gets the name of the supervisors associated with the user ID numbers.
```

Comment:
IDs should never go to the browser, so this endpoint needs changing to not use an ID number.

Also, I can't tell if this gets the IDs of the supervisors for one user's jobs, or does this get the name of a person (who is a supervisor) with a given ID?

Singular and plural should match:
"names" plural names for plural supervisors, or "name" for single supervisor. 

"Student" (to distinguish from staff)